source/form.html

```diff
@@ -93,6 +93,68 @@ <h1 class="contest__title">
           путешествие с Пинк
         </h1>
 
+        <p class="contest__info">
+          Поделитесь своей историей и получите шанс
+          выиграть ценный приз — 1000
+          миль на вашу бонусную карту!
+          Пожалуйста, заполните форму ниже:
+        </p>
+
+        <form class="form" action="#" method="GET">
+
+          <fieldset class="form__name">
+            <legend class="visually-hidden">Фамилия, имя, отчество</legend>
+            <label class="form__text-label" for="surname">Фамилия:</label>
+            <input class="form__text-input" type="text" name="Фамилия" placeholder="Укажите фамилию *" required id="surname">
+            <label class="form__text-label" for="name">Имя</label>
+            <input class="form__text-input" type="text" name="Имя" placeholder="Введите ваше имя *" required id="name">
+            <label class="form__text-label" for="patronymic">Отчество</label>
+            <input class="form__text-input" type="text" name="Отчество" placeholder="Ну и отчество тоже" id="patronymic">
+          </fieldset>
+
+          <fieldset class="form__app">
+            <legend class="form__title">С каким приложением путешествовали?</legend>
+            <input class="form__check-input" checked type="radio" name="app" id="ios">
+            <label class="form__check-label" for="ios">Pink для iOS</label>
+            <input class="form__check-input" type="radio" name="app" id="android">
+            <label class="form__check-label" for="android">Pink для Android</label>
+            <input class="form__check-input" type="radio" name="app" id="windows">
+            <label class="form__check-label" for="windows">Windows Phone</label>
+          </fieldset>
+
+          <fieldset class="form__contact">
+            <legend class="form__title">Контактная информация</legend>
+            <input class="form__text-input" type="tel" name="Телефон" placeholder="Номер, пожалуйста" id="tel">
+            <label class="form__text-label" for="tel">Номер телефона</label>
+            <input class="form__text-input" type="email" name="Почта" placeholder="Введите почту *" required id="email">
+            <label class="form__text-label" for="email">Адрес почты</label>
+          </fieldset>
+
+          <fieldset class="form__achive">
```

Comment:
```suggestion
          <fieldset class="form__achievements">
```

Множественное число в названии классов указывает на разнородные перечисление внутри блока.